scripts/deeplink-dispatcher.sh

```
@@ -0,0 +1,316 @@
+    fi
+  done
+}
+
+# Function to send link to selected Android Emulators
+send_to_android() {
+  local link="$1"
+  shift
+  local emulators=("$@")
+  
+  for emu in "${emulators[@]}"; do
+    local emu_name=$(echo "$emu" | cut -d':' -f1)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)